src/charm.py

```diff
@@ -170,7 +167,7 @@ def _create_external_upf_service(self) -> None:
             ),
         )
 
-        client.apply(service, field_manager=DEFAULT_FIELD_MANAGER)
+        client.apply(service, field_manager=self.__class__.__name__)
```

Comment:
While there's nothing wrong with this, did you consider using `self.app.name` here? 
I'm not sure if this will be visible to the user anywhere, but if yes, perhaps pointing to the application name would be slightly better than to the class name?
WDYT?

Reply:
The user could see the manager by using --show-managed-fields option if desired. 
Using the app name makes sense to me.

Reply:
Done